knowledge-base/ChrisCarini/intellij-platform-plugin-verifier-action/action-security.yml

```diff
@@ -0,0 +1,6 @@
+name: 'IntelliJ Platform Plugin Verifier'
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
+    contents: read
+    contents-reason: to avoid github rate limiting #Reference: https://github.com/ChrisCarini/intellij-platform-plugin-verifier-action#github-token-authentication
```

Comment:
Actually if this is for rate limiting only, we should not need `contents: read`. It should only need metadata permissions, which are already there. So, you can remove the permissions part...